README.md

```diff
@@ -368,6 +381,20 @@ c2patool sample/C.jpg trust \
 
 **Note:** This sample image should show a `signingCredential.untrusted` validation status since the test signing certificate used to sign them is not contained on the trust lists above.
 
+## Adding a manifest to fragmented BMFF content
+
+The ISO base media file format (BMFF) is a container file format that defines a structure for files that contain time-based multimedia data such as video and audio.
+
+Add a manifest to fragmented BMFF by using the `fragment` subcommand, as follows:
+
+```
+c2patool <PATH> fragment [--fragments_glob <PATTERN>]
+```
+
+### Additional option
+
+The `--fragments_glob` option is only available with the `fragment` subcommand and specifies the glob pattern to find the fragments of the asset. The path is automatically set to be the same as the "init" segment.
```

Comment:
The path is automatically set to be the same as the "init" segment so the pattern should only match segment file names.